README.md

```diff
@@ -83,6 +83,7 @@ export LIBVA_DRIVER_NAME=iHD
 * WHL (Whiskey Lake)
 * CML (Comet Lake)
 * ICL (Ice Lake)
+* TGL (Tiger Lake)
```

Comment:
Do we support EHL in the driver? If yes, do we need to mention it separately in the whole readme?

Reply:
I would like to merge KBL/CFL/WHL/CML to one column KBL+ and all derivative platform should be just a explanation

Reply:
Yep, this makes sense for the tables below. They become too wide. Though I would list everything in this list here one by one to make sure people understand we support derivative platforms.